goguen/mantis.nix

```diff
@@ -1,20 +1,20 @@
-{ stdenv, pkgs, mantisSrc, sbtVerify, protobuf }:
+{ stdenv, pkgs, getSrc, sbt-verify, protobuf }:
 
-stdenv.mkDerivation {
-  name = "mantis";
-  src = mantisSrc;
+stdenv.mkDerivation rec {
+  name = "mantis-cardano";
+  src  = getSrc name;
 
-  buildInputs = with pkgs; [ scala sbt sbtVerify unzip protobuf openjdk8 ];
+  buildInputs = with pkgs; [ scala sbt sbt-verify unzip protobuf openjdk8 ];
 
   outputs = [ "out" "zip" ];
 
   configurePhase = ''
     export HOME="$NIX_BUILD_TOP"
     export "_JAVA_OPTIONS=-Dsbt.global.base=.sbt/1.0 -Dsbt.ivy.home=.ivy"
 
-    cp -r ${sbtVerify}/.ivy .
-    cp -r ${sbtVerify}/.sbt .
-    cp -r ${sbtVerify}/target .
+    cp -r ${sbt-verify}/.ivy .
+    cp -r ${sbt-verify}/.sbt .
+    cp -r ${sbt-verify}/target .
```

Comment:
you can also do `cp -r ${sbr-verify}/{.ivy,.sbt,target} .`

Reply:
I'm looking towards removing the initial G-ala-C implementation at some point in the future.
We can't do it for a reason right now.
In this light, I'm not sure it'd be an economic expenditure of effort to focus on making this pretty.